buildSrc/src/main/groovy/swatch.quarkus-rest-client-conventions.gradle

```diff
@@ -45,6 +45,7 @@ openApiGenerate {
 dependencies {
     implementation enforcedPlatform(libraries["quarkus-bom"])
     api 'io.quarkus:quarkus-rest-client-reactive-jackson'
+    api libraries["jackson-databind-nullable"]
```

Comment:
I had to add this dependency to avoid having the following compilation failure:
```
/home/jcarvaja/sources/RedHatInsights/rhsm-subscriptions/clients/swatch-internal-subscription-client/build/generated/src/main/java/com/redhat/swatch/clients/swatch/internal/subscription/api/model/RpcResponse.java:22: error: package org.openapitools.jackson.nullable does not exist
import org.openapitools.jackson.nullable.JsonNullable;
```

This dependency is in the spring rest-client conventions as well.